src/Libraries/CoreNodeModels/DynamoConvert.cs

```diff
@@ -75,6 +79,7 @@ public ConversionUnit SelectedFromConversion
             }
         }
 
+        [JsonProperty("ToConversion")]
```

Comment:
I don't know why we would allow these to be set differently. I propose that we consolidate them into one conversion factor.